package.json

```diff
@@ -37,31 +37,30 @@
         "license:check": "license-check-and-add check -f license-config.json",
         "license:add": "license-check-and-add add -f license-config.json",
         "license:remove": "license-check-and-add remove -f license-config.json",
-        "preinstall": "npx only-allow pnpm",
```

Comment:
good point,we should only added this for internal packages or mainly workspace roots, and never to published packages.